lib/statusCodeIsOk.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+var dateLog = require('./dateLog');
+/** Helper to check statusCode
+ *
+ * @param {Object} response JSON response
+ * @return {boolean} true if the statusCode is 200
+ */
+module.exports = function statusCodeIsOk(response) {
```

Comment:
There are a few issues with this function.  status can be any 2XX number, so we need to check more than 200.  Logging everything every time is probably overkill as well.  We are also duplicating checking because we construct a Promise.reject if this function returns false.  You can wrap this all together and make better use of promises with something like the below:

```javascript
function checkStatus(url, response) {
    if (response.status < 200 || response.status >= 300) {
       return Promise.reject('Status code ERROR: ' + url + ' ' + response.statusCode + ', ' + response.statusMessage);
    }
    return Promise.resolve(response);
}
```

Then you use it with 

```javascript
return requestPromise.get({
    ...
    })
    .then(function (response) {
        return checkStatus(response);
    }
    .then(function(response) {
      // We only end up here if response was valid
    }
```